local/flake.nix

```diff
@@ -0,0 +1,51 @@
+{
+  description = "Local Development Environment";
+  inputs.nosys.url = "github:divnix/nosys";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+  inputs.devshell.url = "github:numtide/devshell";
+  inputs.call-flake.url = "github:divnix/call-flake";
+  outputs = inputs @ {
+    nosys,
+    call-flake,
+    ...
+  }:
+    nosys ((call-flake ../.).inputs // inputs) (
+      {
+        self,
+        nixpkgs,
+        devshell,
+        std,
+        ...
+      }:
+        with nixpkgs.legacyPackages;
+        with nixpkgs.legacyPackages.nodePackages;
+        with devshell.legacyPackages; let
+          inherit (lib.stringsWithDeps) noDepEntry;
+          checkMod = {
```

Comment:
I see what it contains but I have no idea what does this name mean. Please rename it

Reply:
Addressed in 93cd451b3bf08ad132da0e18839473efda672658